rollback_on_fail.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+rollback () {
+    wget https://github.com/Jellyfish-AI/jf_agent/archive/refs/tags/stable.tar.gz
+    tar -zxvf stable.tar.gz --directory ./
```

Comment:
```suggestion
    curl -L -o - https://github.com/Jellyfish-AI/jf_agent/archive/refs/tags/stable.tar.gz | tar -xzv --directory ./
```
This doesn't even write the temporary file